server/src/test/java/io/druid/server/coordinator/rules/LoadRuleTest.java

```diff
@@ -213,6 +216,147 @@ public boolean appliesTo(Interval interval, DateTime referenceTimestamp)
   }
 
   @Test
+  public void testLoadPriority() throws Exception
+  {
+    final ReplicationThrottler limitedThrottler = new ReplicationThrottler(1, 1);
```

Comment:
More tests would be good, also it's usually better the mock environment creation to look concise,     so ideal unit tests should contain initialization/asserts code in the ratio 1/3.